spec/ims/lti/models/messages/registration_request_spec.rb

```diff
@@ -45,7 +46,7 @@ module IMS::LTI::Models::Messages
       end
 
       it 'returns the key and secret' do
-        expect(subject.generate_key_and_password).to eq 2.times.map{ 'random_uuid' }
+        expect(subject.generate_key_and_password).to eq 2.times.map {'random_uuid'}
```

Comment:
I don't mind the space after `map` however, it's nice to have the whitespace between the string and brackets. I lean on the Javascript preference of it.